lib/Driver/Driver.cpp

```diff
@@ -1778,6 +1781,16 @@ void Driver::buildOutputInfo(const ToolChain &TC, const DerivedArgList &Args,
 
   }
 
+  if (const Arg *A = Args.getLastArg(options::OPT_lto)) {
+    if (A->containsValue("llvm"))
```

Comment:
I think that you should be more stringent with the check here: `prefixllvmsuffix` is not a valid value for `lto`.  In fact, you should emit a diagnostic for an invalid argument instead.

Reply:
Changed to use `StringSwitch` to match exactly. 👍